tests/chaos/utils.py

```
@@ -415,3 +416,176 @@ def get_instance_type(name):
+            LOGGER.info("Pod deleting process finished (max_duration reached)")
+            break
+
+        try:
+            delete_pods(
+                client=client,
+                pod_prefix=pod_prefix,
+                namespace_name=namespace_name,
+                ratio=ratio,
+            )
+        except ResourceNotFoundError:
+            LOGGER.info(f"No pods found with prefix {pod_prefix} in this iteration")
```

Ruff (0.15.0)

[warning] Logging statement uses f-string

(G004)